ci-operator/step-registry/openshift/e2e/gcp/custom-endpoints/openshift-e2e-gcp-custom-endpoints-workflow.yaml

```diff
@@ -0,0 +1,29 @@
+workflow:
+  as: openshift-e2e-gcp-custom-endpoints
+  steps:
+    pre:
+    - ref: gcp-provision-minimal-permission
+    - ref: gcp-provision-vpc
+    - ref: gcp-provision-custom-endpoints
```

Comment:
If the cluster is going to use private endpoint, does it make more sense in a disconnected environment to ensure the cluster is using the private endpoints, but not still using the origin endpoints ?

If we remove `--enable-private-ip-google-access` on the [line](https://github.com/openshift/release/blob/master/ci-operator/step-registry/gcp/provision/vpc/gcp-provision-vpc-commands.sh#L64) but enable private endpoints, can the installation still work well ?